public/js/custom-wt.js

```diff
@@ -1,25 +1,22 @@
 jQuery(function($) {
   $(document).ready(function () {
-    $('.gp-email').on('click', function () {
-      Webtrends.multiTrack({argsa: ['DCSext.CTSLinkClicks', 'GP-Email', 'WT.dl', '121']});
-    });
-    $('.gp-book-online').on('click', function () {
-      Webtrends.multiTrack({argsa: ['DCSext.CTSLinkClicks', 'GP-Booking', 'WT.dl', '121']});
-    });
-    $('.repeat-prescription-online').on('click', function () {
-      Webtrends.multiTrack({argsa: ['DCSext.CTSLinkClicks', 'GP-Repeat-Prescription', 'WT.dl', '121']});
-    });
-    $('.coded-records-online').on('click', function () {
-      Webtrends.multiTrack({argsa: ['DCSext.CTSLinkClicks', 'GP-Medical-Record', 'WT.dl', '121']});
-    });
-    $('.gp-patient-survey').on('click', function () {
-      Webtrends.multiTrack({argsa: ['DCSext.CTSLinkClicks', 'GP-Patient_Survey', 'WT.dl', '121']});
-    });
-    $('.gp-website').on('click', function () {
-      Webtrends.multiTrack({argsa: ['DCSext.CTSLinkClicks', 'GP-Website', 'WT.dl', '121']});
-    });
-    $('.gp-website-opening-times').on('click', function () {
-      Webtrends.multiTrack({argsa: ['DCSext.CTSLinkClicks', 'GP-Website_OpeningTimes', 'WT.dl', '121']});
+
+    var anchors = {
+      'gp-email' : 'GP-Email',
+      'gp-book-online' : 'GP-Booking',
+      'repeat-prescription-online' : 'GP-Repeat-Prescription',
+      'coded-records-online' : 'GP-Medical-Record',
+      'gp-patient-survey' : 'GP-Patient_Survey',
+      'gp-website-register' : 'GP-Register_Patient',
+      'gp-website-register-other' : 'GP-Register_Other',
+      'gp-website' : 'GP-Websitet',
+      'gp-website-opening-times' : 'GP-Website_OpeningTimes'
+    }
+
+    $.each(anchors , function(prop, val) {
```

Comment:
I couldn't get the `forEach(...)` to work. Happy for you to take a look and show me how you did it.